consoles/sshVirtsh.pm

```diff
@@ -34,8 +34,8 @@ sub new {
     my $self = $class->SUPER::new($testapi_console, $args);
 
     $self->instance(get_var('VIRSH_INSTANCE', 1));
-    # default name
-    $self->name("openQA-SUT-" . $self->instance);
+    # default name is openQA-SUT-1
+    $self->name(get_var('VIRSH_GUESTNAME', "openQA-SUT-") . $self->instance);
```

Comment:
I have a mixed feelings about the separator (-) to be a part of VIRSH_GUESTNAME.

@mimi1vx What separator are those test machines using? Can the dash be hardcoded?
```perl
$self->name(get_var('VIRSH_GUESTNAME', 'openQA-SUT') . '-' . $self->instance);
```
or
```perl
$self->name(sprintf('%s-%u', get_var('VIRSH_GUESTNAME', 'openQA-SUT'), $self->instance));
```